packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Configures whether [SnackBar] should be fixed to the bottom or floating
```

Comment:
Consider specifiying if the floating is in the z or y access. For this component, it's a little confusing.

Reply:
done